step.yml

```diff
@@ -1,8 +1,30 @@
 title: "Android Sign"
-summary: Android Sign
-description: 
-  With this step, you can digitally sign your app's APK file in your workflow. 
-  All you have to do is upload your keystore file in the Code signing tab of the Workflow Editor.
+summary: Signs your APK or Android App Bundle before uploading it to Google Play Store.
+description: |-
+  Once you have uploaded your keystore file and provided your keystore credentials on the **Code Signing** tab of the Workflow Editor, the **Android Sign** Step signs your APK digitally. Bitrise assigns Environment Variables to the uploaded file and credentials, and uses those in the respective fields of the **Android Sign** Step. Once the Step runs, it produces a signed APK or App Bundle which will be used as the input value of the **App file path** field in the **Google Play Deploy** Step.   
+
+  ### Configuring the Step
+
+  1. Add the **Android Sign** Step after a build Step in your deploy workflow.
+  2. Upload the keystore file to the **Upload file** field on the **Code Signing** tab.
+  3. Provide your keystore password, keystore alias and private key password to the relevant fields on the **Code Signing** tab.
+  4. Run your build.
+
+
+  ### Troubleshooting
+  Make sure you have the **Android Sign** Step right after a build Steps but before **Deploy to Google Play** Step in your deploy workflow.
+  If you wish to get your Android project signed automatically, use the **Android Sign** Step and do not set any gradle task for the signing, otherwise, the Step will fail.
+
+
+  ### Useful links
+  [Android code signing using Android Sign Step](https://devcenter.bitrise.io/code-signing/android-code-signing/android-code-signing-using-bitrise-sign-apk-step/)
+  [Android deployment](https://devcenter.bitrise.io/deploy/android-deploy/android-deployment-index/)
+
+
+  ### Related Steps
+  [Android Build](https://www.bitrise.io/integrations/steps/android-build)
+  [Gradle Runner](https://www.bitrise.io/integrations/steps/gradle-runner)
+  [Deploy to Bitrise.io](https://www.bitrise.io/integrations/steps?query=deploy%20to%20bitrise)
```

Comment:
Please change the URL to be consistent with other ones: https://www.bitrise.io/integrations/steps/deploy-to-bitrise-io

Reply:
👍